files/default/sql/ddl/updates/3.1.0.sql

```diff
@@ -67,3 +67,48 @@ CREATE TABLE IF NOT EXISTS `pki_serial_number` (
 
 DROP TABLE `dela`;
 DROP TABLE `hopssite_cluster_certs`;
+
+-- rename transformation functions output types
+UPDATE transformation_function
```

Comment:
This doesn't work. MySQL has an option called `safe_update` (see doc: https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_safe-updates) - where if you try to run a update statement without a `WHERE` condition on a key, it's going to fail.

It works if you try the update manually as you run it as root and safe update is disabled, but when we run the migration with flyway, we need to disable safe updates before doing this kind of migrations.

See example here: https://github.com/logicalclocks/hopsworks-chef/blob/f178c2b8a312b568fec9df4e4d8ce66ec969f38c/files/default/sql/ddl/updates/3.0.0.sql#L151

Also make sure you re-enable it afterwards.

Reply:
Oh, thanks for the hint. It indeed worked just fine when I tested it manually.